monero-outreach-docs/en/archive/monero_means_money_en.md

```diff
@@ -0,0 +1,40 @@
+# Monero Means Money Movie Removed From #1 List
+
+**“The Top Movie in America Last Weekend Was about privacy...briefly.” - 21th of April, 2020**
+
+Amid warnings on government privacy intrusions during the COVID-19 pandemic, the number-one movie in America — if only briefly — was about privacy. It temporarily had the sales to be #1 as its weekend sales exceeded the entire weekend (4/10-4/12) box office of other movies. It was listed as #1 for a couple hours and then was promptly removed from the movie industry data website [The Numbers](https://web.archive.org/web/20200413125904/https://www.the-numbers.com/weekend-box-office-chart).
```

Comment:
The final part of this text seem wrong. There is a dot missing and there are two URLs in one.

Reply:
Hey Erciccione, where is the dot missing? Check the link, that's how it displays the Weekend domestic Chart for April 10, 2020.

Reply:
Yes you are right. I didn't realize it was an archive.org link.

Reply:
It's okay :)